Public/Src/Deployment/buildXL.dsc

```diff
@@ -23,6 +23,13 @@ namespace BuildXL {
             importFrom("BuildXL.Tools").BxlScriptAnalyzer.exe,
             importFrom("BuildXL.Cache.VerticalStore").Analyzer.exe,
 
+            // content placement
+            ...addIfLazy(qualifier.targetFramework === "net472" && qualifier.targetRuntime !== "osx-x64", () => [
+                importFrom("BuildXL.Tools").Tools.ContentPlacement.Extraction.exe,
+                importFrom("BuildXL.Tools").Tools.ContentPlacement.Extraction.buildDownloaderLogFile,
+                importFrom("BuildXL.Tools").Tools.ContentPlacement.Extraction.staticResources,

```

Comment:
If you add these to the `runtimeContent` field when compiling the executable they will be automatically copied whereever the exe is deployed, like for unittests etc, so not everybody that lays out this assemlby has to deal with copying the extra files.